tests/Unit/Evolution/EventsAndDenseTriggers/DenseTriggers/Test_Filter.cpp

```diff
@@ -0,0 +1,153 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
+#include <memory>
+#include <sstream>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Evolution/EventsAndDenseTriggers/DenseTrigger.hpp"
+#include "Evolution/EventsAndDenseTriggers/DenseTriggers/Filter.hpp"
+#include "Evolution/EventsAndDenseTriggers/DenseTriggers/Times.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/Evolution/EventsAndDenseTriggers/DenseTriggers/TestTrigger.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Trigger.hpp"
+#include "Time/Tags.hpp"
+#include "Time/TimeStepId.hpp"
+#include "Time/Triggers/TimeCompares.hpp"
+#include "Utilities/Registration.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+template <typename RegistrarList>
+class TestTrigger : public Trigger<RegistrarList> {
+ public:
+  /// \cond
```

Comment:
does this need to be updated with the recent checks for `cond` in cpp files?

Reply:
Done.